### ISSUE TITLE
The train loss not change

### ISSUE THREAD

**Issue by trustguan** [xingyizhou/CenterNet#798]
Thanks to  your great job.
When I train the network use command:
python main.py ctdet --exp_id coco_dla_2x --batch_size 128 --lr 1.25e-4 --gpus 0 --num_workers 0  --val_intervals 5 --num_epochs 100  --master_batch 15

The train loss not change like it:

`ctdet/coco_dla_2x |################################| val: [70][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.021s(0.022s) |Net 0.064s
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x |################################| val: [75][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.022s(0.022s) |Net 0.064s
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x |################################| val: [80][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.019s(0.021s) |Net 0.062s
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x |################################| val: [85][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.035s(0.024s) |Net 0.068s
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x |################################| val: [90][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.023s(0.023s) |Net 0.066s
Drop LR to 1.25e-05
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x |################################| val: [95][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.020s(0.022s) |Net 0.065s
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x
ctdet/coco_dla_2x |################################| val: [100][10/11]|Tot: 0:00:00 |ETA: 0:00:01 |loss 30.4600 |hm_loss 28.3072 |wh_loss 16.6406 |off_loss 0.4887 |Data 0.022s(0.022s) |Net 0.063s`

What is the problem?

**Comment by CH3OCH3-lio** [comment]
I have the same problem as you, have you solved this problem?

**Comment by CConory** [comment]
this may be caused by the big learning rate

**Comment by CH3OCH3-lio** [comment]
what may be the suitable leraning rate?

**Comment by CH3OCH3-lio** [comment]
学习率多大合适 哥们

**Comment by CConory** [comment]
> 

depend on your input shape, network architecture, and so on...
could use a very tiny learning rate, such as 1e-5, then increase step by step to choose the best lr.
Or change the optimizer.
I guess this may be caused by the backward in the sigmoid opt of the hm branch's .

**Comment by Juphex** [comment]
I had the same problem. The learning rate that you use was in my case way too low. I had to change it to 1.25e-3 and also 1.25e-2 works. 
However, learning rates lower than that useless regarding the wh_loss.
